pages/api/slider.md

```diff
@@ -41,23 +47,19 @@ Any other properties supplied will be provided to the root element (native eleme
 You can override all the class names injected by Material-UI thanks to the `classes` property.
 This property accepts the following keys:
 
-
-| Name | Description |
-|:-----|:------------|
-| <span class="prop-name">root</span> | Styles applied to the root element.
-| <span class="prop-name">container</span> | Styles applied to the container element.
-| <span class="prop-name">track</span> | Styles applied to the track elements.
-| <span class="prop-name">trackBefore</span> | Styles applied to the track element before the thumb.
-| <span class="prop-name">trackAfter</span> | Styles applied to the track element after the thumb.
-| <span class="prop-name">thumbWrapper</span> | Styles applied to the thumb wrapper element.
-| <span class="prop-name">thumb</span> | Styles applied to the thumb element.
-| <span class="prop-name">thumbIconWrapper</span> | Class applied to the thumb element if custom thumb icon provided.
-| <span class="prop-name">thumbIcon</span> | 
-| <span class="prop-name">disabled</span> | Class applied to the track and thumb elements to trigger JSS nested styles if `disabled`.
-| <span class="prop-name">jumped</span> | Class applied to the track and thumb elements to trigger JSS nested styles if `jumped`.
-| <span class="prop-name">focused</span> | Class applied to the track and thumb elements to trigger JSS nested styles if `focused`.
-| <span class="prop-name">activated</span> | Class applied to the track and thumb elements to trigger JSS nested styles if `activated`.
-| <span class="prop-name">vertical</span> | Class applied to the root, track and container to trigger JSS nested styles if `vertical`.
+- `root`
```

Comment:
Was this intended? The classes don't have descriptions in `Slider.js` either

Reply:
Oops. I'm fixing it.